src/routes/articles.js

```diff
@@ -37,7 +37,8 @@ const transformArticleResponse = (article) => ({
 	published: article.published,
 	createdOn: article.createdAt,
 	articleId: article.id,
-	tagId : article.tagId
+	tagId : article.tagId,
+	type : article.type
```

Comment:
we won't need type in the response, its just an internal thing. the endpoint already is just for `/articles`

Reply:
alright thanks